openreview/conference/matching.py

```diff
@@ -182,7 +183,12 @@ def _build_conflicts(self, submissions, user_profiles):
                         writers=[self.conference.id],
                         signatures=[self.conference.id]
                     ))
-            openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        openreview.tools.post_bulk_edges(client=self.client, edges=edges)
+        # Perform sanity check
+        edges_iter = openreview.tools.iterget_edges(self.client, invitation=invitation.id)
+        edges_posted = sum(1 for _ in edges_iter)
```

Comment:
we usually convert the iterator to a list:

```suggestion
        edges_posted = len(list(edges_iter))
```